core/serialisation/data_serialiser.py

```diff
@@ -78,6 +80,9 @@ def serialise_download_data(
     for sheet in sheets_required:
         query_extender, schema = table_queries[sheet]
         extended_query = query_extender(base_query)
+        # TODO: remove this conditional test on postgres once we strip out sqlite (see FMD-8)
+        if db.engine.name == "postgresql":
+            db.session.execute(text("SET LOCAL work_mem TO '128MB'"))
```

Comment:
Does this not need to be reset as per the linked blog post?
> For the targeted queries, set the optimal value at the session level, then make sure to reset it (RESET work_mem;) back to the instance level setting at the end of the request so that the memory is released back to system and used for other activities.

Reply:
Change made as per suggestions

Reply:
I'm not 100% sure that line in the blog is technically correct, I thought the session is handled as a context, so will be garbage collected anyway between each query? Otherwise we would definitely have run out of memory overall

Reply:
But maybe just worth double checking :)